packages/monaco/src/code-editor.tsx

```diff
@@ -5,6 +5,7 @@ import { useTheme } from 'next-themes';
 import { useMemo } from 'react';
 import { useEditorSettingsStore } from './settings-store';
 import { libSource } from './editor-types';
+import type * as MonacoEditor from 'monaco-editor';
```

Comment:
do you need the type keyword in addition to typeof of below?

Reply:
Technically we don't but Eslint will throw a warning:
"All imports in the declaration are only used as types. Use `import type`."

Reply:
ah ok it’s ok to eslint ignore when needed

Reply:
ok done